src/runtime_src/core/tools/xbmgmt2/OO_NIFD.cpp

```diff
@@ -49,7 +141,37 @@ OO_NIFD::OO_NIFD( const std::string &_longName)
 void
 OO_NIFD::execute(const SubCmdOptions& _options) const
 {
-  _options.size();
-  printHelp();
-}
+  XBU::verbose("SubCommand option: nifd");
+  XBU::verbose("Option(s):");
+  for (auto & aString : _options) {
+    std::string msg = "   ";
```

Comment:
Since this code is simply add some spaces in front of the string, can you refactor it to:
1) Either use a boost::format(), which might be too much.
OR use
2) XBU::verbose(std::string("   ") + aString);